### PR TITLE
Add `filter` and `offset-path` to list with URLs

### PR DESCRIPTION
**Pull request by brettz9**
Add `filter` and `offset-path` to list with URLS

Also alphabetizes.

> What was wrong/why is this fix needed? (quick summary only)

Missing some items, and hard to parse when not alphabetical.

> MDN URL of the main page changed

https://developer.mozilla.org/en-US/docs/Web/CSS/url()

> Issue number (if there is an associated issue)

N/A

> Anything else that could help us review it

### CODE REVIEW

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
@brettz9 it’s great to see you (back) here — thanks much, and congrats on landing your first docs change here under the new system — welcome aboard 🎉

**Comment by brettz9** [comment]
@sideshowbarker : Thank you for the kind words and welcome!

Btw, I think it's an interesting approach to work with Git files, and good to see there's a preview opportunity. 

I'd be great if this approach could be used with any Markdown+HTML project in a reliable way. A pity https://dynalon.github.io/mdwiki/#!quickstart.md looks like it was abandoned, but it'd be great to see some such fork, (though I can imagine MDN has quite a few of its own needs to make such a proposed adoption not an easy conversion).

**Comment by sideshowbarker** [comment]
As far Markdown goes, all of the MDN content will be getting migrated to Markdown within a reasonable amount of time. Quite a bit of the preparation for the migration is already done. So the sources are eventually going to be nearly completely HTML-free.

I think a lot of users are going to find it pretty easy and appealing to edit the sources as Markdown directly in the GitHub UI — and not need to deal with the extra complications of trying to get the HTML right. I think once that migration has taken place, we’re going to end up with something that works a lot better for users then it did for us and them back when we edited it all in the wiki.

**Comment by brettz9** [comment]
Sure, it indeed sounds like a good approach, and good to know about moving more to Markdown.

If you get to something like that, I just meant to suggest it might be worth looking into using, developing, or recommending a local WYSIWYG Markdown tool, to fully replicate the old functionality and improve on it (e.g., so that one could, for example, add "orange" links to page not yet created, then easily right-click them (or existing "blue" links) to actually go to edit the source). I think the community is oddly lacking such good software, so it would be useful beyond the project as well, I think. Anyways, just throwing it out there for what it may be worth.

**Comment by Ryuno-Ki** [comment]
@brettz9 I encourage you to open a new discussion with these ideas.
I am afraid, they would get lost in a merged PR thread.
